onnxruntime/core/session/custom_ops.cc

```diff
@@ -106,6 +106,17 @@ ORT_API_STATUS_IMPL(OrtApis::KernelContext_GetGPUComputeStream, _In_ const OrtKe
   API_IMPL_END
 };
 
+ORT_API_STATUS_IMPL(OrtApis::KernelContext_GetAllocator, _In_ const OrtKernelContext* context, _In_ const OrtMemoryInfo* mem_info, _Outptr_ OrtAllocator** out) {
+  API_IMPL_BEGIN
+  onnxruntime::AllocatorPtr allocator = reinterpret_cast<const onnxruntime::OpKernelContextInternal*>(context)->GetAllocator(*mem_info);
+  if (!allocator) {
+    return OrtApis::CreateStatus(ORT_INVALID_ARGUMENT, "No requested allocator available");
+  }
+  *out = new onnxruntime::OrtAllocatorImplWrappingIAllocator(std::move(allocator));
```

Comment:
>new [](http://example.com/codeflow?start=9&length=3)

raw "new" is going to trigger warnings in static analysis. usually rely on unique_ptr for creations, and use .release() to acquire the raw thereafter.

Reply:
It follows the implementation of CreateAllocator in allocator_adapters.cc. Shall we follow the convention by adding 

#if defined(_MSC_VER) && !defined(__clang__)
#pragma warning(disable : 26409)
#endif

to suppress the warning?